### ISSUE TITLE
If it's possible to add ruler line in blank lines

### ISSUE THREAD

**Issue by flynnoct** [freakone/cursoruler#6]
Hey freakone,

Love your cursoruler! I have been finding a vertical highlight plugin for VSCode for a long time. Fortunately I got your extension :). And may I make a tiny request for it? 

When blank lines exists in the editor, cursoruler will "skip" these lines (as line 3, 8 and 11 showed). If it's possible for you to make the ruler-line **continuous**?

I will be grateful if you can do this, thanks for providing us such a good plugin.

<img width="410" alt="Screen Shot 2021-11-30 at 17 53 02" src="https://user-images.githubusercontent.com/43941314/144024944-25e6fb3c-555b-469e-909c-a284052797bb.png">

**Comment by shwivel** [comment]
I agree. This is perhaps the biggest reason I still use Sublime. I don't know why Microsoft doesn't see the value in this but thank you for your efforts with this extension. Unfortunately, it falls a bit short in this way, compared to the CursorRuler extension in Sublime (https://github.com/icylace/CursorRuler). The visual aid of having a full ruler is quite helpful. Not so much when it is broken up into pieces, as depicted in the above post.
